_component_design/table.md

```diff
@@ -0,0 +1,343 @@
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Spacing
+
+Refer to these spacing specifications when you construct a table header.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Table header spacing"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/tables/table/placeholder-image.png" width="444"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Table row specifications
+
+The table row contains data associated with an object.
+
+Use the following specifications when you construct a table row.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+A table row is comprised of the following elements:
+
+- **Select checkbox (Required/Optional)**: Include a checkbox if a user needs to select or manipulate data within a table. Always list the checkbox as the first element in the table row. See [table controls]({{site.baseurl}}/components/tables-controls.html). The user clicks the checkbox and then clicks a button or selects an action from a cog.
+- **Summary rows (Required/Optional)**: You can add a summary row at the bottom of the table that displays column summary information such as totals.
+- **Bulk selection (Required/Optional)**: Include bulk selection when you want the user to select or deselect all objects in the table.
+- **Totaling (Required/Optional)**: Totaling is the sum of all column summaries. Totaling is used to calculate a grand total.
+- **Reveal (Required/Optional)**: Some table rows can contain a lot of information, including data and text. If you construct a table row with reveal, then the user can expand or collapse the row, which hides or shows more data. By providing the ability to hide information, reveal allows for a more optimum use of table space.  
+- **Heartbeat (Required/Optional)**: When the user edits an object in the table, the edited rows glows briefly (the *heartbeart*) when the user saves the change. As the table refreshes, the heartbeat focuses the users' eyes on the change they made.
+- **Cog (Required/Optional)**: The cog contains a list of actions from which the user can select. Include the action cog menu if the objects in each row have two or more available actions.
+- **Status indicator (Optional)**: If the objects in the table are services, use a status indicator icon to communicate service status.
```

Comment:
cut the status indicator here.

It needs to be considered more thoroughly before we include it. we'll look at it more with the data viz work.  and maybe we will add it to tables once we have a firmer opinion/style for it.

Reply:
sounds good!